src/js/components/ButtonLabel.js

```diff
@@ -0,0 +1,27 @@
+// (C) Copyright 2014-2015 Hewlett Packard Enterprise Development LP
+
+import React, { Component, PropTypes } from 'react';
+
+export default class ButtonLabel extends Component {
```

Comment:
Since we didn't publish the old Label component on grommet.io, I think we can remove it, perhaps adding a note if they pass a property that was in the old one but not in the new one.